dist/gcc-compatible/Hacl_Hash_Blake2.h

```diff
@@ -47,76 +46,6 @@ extern "C" {
 
 typedef uint8_t Hacl_Impl_Blake2_Core_m_spec;
 
-uint64_t Hacl_Hash_Core_Blake2_update_blake2s_32(uint32_t *s, uint64_t totlen, uint8_t *block);
```

Comment:
all of these were previously exposed (in spite of being private) because Hacl_Streaming_Blake2 needed to "see" those variants -- now they're only in the internal header